README.md

```diff
@@ -1,2 +1,41 @@
-# HW01
-Edit the readme
+### HW01
+
+*** 
```

Comment:
I like these horizontal lines. I didn't know you can add them by using the three asterisks, but I think it's a great way to divide the documents into sections.